lib/app/bloc/app_bloc.dart

```diff
@@ -32,6 +32,8 @@ class AppBloc extends Bloc<AppEvent, AppState> {
       yield _mapUserChangedToState(event, state);
     } else if (event is AppLogoutRequested) {
       _authenticationRepository.logOut();
+    } else if (event is AppUserDeleteRequested) {
+      _authenticationRepository.deleteUser();
```

Comment:
Mozna zamienic else if na returny, nie pamietam jak to sie dokladnie nazywa. Ale Dawid kazal mi tak robic i wsm jest czytelniej